src/pages/Content/index.js

```diff
@@ -13,3 +13,17 @@ function click(selector) {
   const element = document.querySelector(selector) || null;
   if (element) element.click();
 }
+// The function plays a recording when needed
+function playRecording(recordedEvents) {
```

Comment:
This should not appear in this branch and PR. Separate the branches.

Reply:
As Agam advised,  can you please remove this function playRecording(recordedEvents) from this branch as it is in another branch?